### DISCUSSION TITLE
Updating beat schedule dynamically

### DISCUSSION THREAD

**Discussion by hben-align** [celery/celery#9084] [Q&A]
Hi,

I'm trying to write an app where the beat schedule is working based on a table in Postgres.

this table can be updated at any time but I don't have to reflect the changes to celery right away (for example I can wait 5 minutes before the changes take effect)

now, I don't see it anywhere but I am wondering if it's possible to use the native beat schedule in this case,
to have a task scheduled to run every 5 minutes and update the beat schedule based on the Postgres table.

is it possible?

I would prefer not to use external modules like redbeat or django, but for that I need to know if its even possible to implement it with the native module

**Comment by thedrow** [comment]
Even if you use an external scheduler, you'll have to restart celery beat for the changes to take effect.
What you're asking is currently not possible.
